source/_integrations/shelly.markdown

```diff
@@ -64,9 +80,12 @@ Names are set from the device web page:
 - Channel name for single-channel devices can be set in **Settings** >> **CHANNEL NAME**
 - Channel name for multi-channel devices can be set in **Settings** >> **CHANNEL NAME** after selecting the channel, by clicking on the channel name.
 
-## Appliance type
+## Entity naming (generation 2)
```

Comment:
We should put this above gen 1 as moving forward, all devices will be gen 2.

Reply:
Actually, I don't think that naming strategy is important to document ? 🤷

Reply:
I believe it will be a long time before Generation 2 becomes popular. But I agree that the entity naming section is not necessary so maybe we should remove it?

Reply:
I think we should keep the entity naming section and for now gen1 above gen2, in the future we can update the documentation and even move all gen1 to the end